sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1995,16 +1995,16 @@ def pct_change_overload(df, periods=1, fill_method='pad', limit=None, freq=None)
     ty_checker = TypeChecker('Method {}().'.format(name))
     ty_checker.check(df, DataFrameType)
 
-    if not isinstance(periods, (types.Integer, types.Omitted)):
+    if not isinstance(periods, (types.Integer, types.Omitted)) and periods != 1:
```

Comment:
Why `periods` could not be equal to other values?

Reply:
@densmirn  ``periods`` could be equal to other values. This condition need that default value supported.